#!/usr/bin/env python
PACKAGE = "interactive_pose_publisher"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("save_path", str_t, 0, "abs_path/filename of yaml to save",  "inital_interactive_pose.yaml")
gen.add("save_pose", bool_t, 0, "Set true to save the current pose in a yaml",  False)
gen.add("marker_size", double_t, 1, "Set the size of the marker", 0.2, 0.01, 1.0)

exit(gen.generate(PACKAGE, "interactive_pose_publisher_node", "interactive_pose_publisher"))
